wp-content/plugins/core/src/Templates/Controllers/Block/Button.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare( strict_types=1 );
+
+namespace Tribe\Project\Templates\Controllers\Block;
+
+use Tribe\Project\Blocks\Types\Button as Button_Block;
+use Tribe\Project\Templates\Components\Link;
+
+class Button extends Block_Controller {
+
+	public function render( string $path = '' ): string {
+		$link = $this->get_link( $this->attributes );
+
+		return $this->factory->get( Link::class, [
+			Link::URL        => $link['url'],
+			Link::CONTENT    => $link['text'],
+			Link::TARGET     => $link['target'],
```

Comment:
Only because I'm ignorant, does link target get populated?

Reply:
There's a UI ticket to add that.